docs/tutorial/using-native-node-modules.md

```diff
@@ -58,10 +58,14 @@ where to download headers and which version to use:
 
 ```bash
 $ cd /path-to-module/
-$ HOME=~/.electron-gyp node-gyp rebuild --target=0.29.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
+$ HOME=~/.electron-gyp node-gyp rebuild --target=0.36.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
+#On Windows Try this
+cd /path-to-module/
+X:\path-to-module>node-gyp rebuild --target=0.36.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
```

Comment:
I don`t think most of windows user use cygwin or bash or powershell, they
just works with plain cmd shell, and they are struggle with the current
tutorial,for me, I tried for two days to find out how to compile it

On Tue, Mar 15, 2016 at 4:16 AM, Luke Page <notifications@github.com> wrote:

> In docs/tutorial/using-native-node-modules.md
> <https://github.com/atom/electron/pull/4726#discussion_r56067994>:
>
> > @@ -58,10 +58,14 @@ where to download headers and which version to use:
> >
> >  ```bash
> >  $ cd /path-to-module/
> > -$ HOME=~/.electron-gyp node-gyp rebuild --target=0.29.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
> > +$ HOME=~/.electron-gyp node-gyp rebuild --target=0.36.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
> > +#On Windows Try this
> > +cd /path-to-module/
> > +X:\path-to-module>node-gyp rebuild --target=0.36.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
>
> Nope it was just an observation that you should say its shell specific not
> os specific
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/atom/electron/pull/4726/files#r56067994>.
>